### PR TITLE
T349806 - Fix Quickstatements image

### PR DESCRIPTION
**Pull request by lorenjohnson**
https://phabricator.wikimedia.org/T349806 and https://phabricator.wikimedia.org/T354260

### CODE REVIEW

**Review by datn — COMMENTED** [review]
Made some clarifying edits and conformed the spelling of QuickStatements

**Comment by lorenjohnson** [comment]
> Made some clarifying edits and conformed the spelling of QuickStatements

Looks good. Thanks and Happy New Year @datn! ✨

File: build/QuickStatements/README.md
Comment (darthmon) [inline build/QuickStatements/README.md:1]: @datn  could you please give a quick check to this documentation changes? 
thanks in advance!

**Review by darthmon — APPROVED** [review]
went through the changes together with Loren

**Pull request merged**